### DISCUSSION TITLE
Making a mechanism to keep track of the pasted stuff..

### DISCUSSION THREAD

**Discussion by fmerch1991** [collove/pasteme#78] [Q&A] [answered]
It would be so lit if you implement some sort of local database to keep the pasted snippets in. Any idea?! Where do users paste their snippets? Ways.

**Comment by lnxpy** [comment, marked as answer]
Thank you for reaching out. Your asking made me open the issue. I had the same idea though. This implementation can be done [here](https://github.com/collove/pasteme-cli/issues/21).

Users use [`pasteme-cli`](https://github.com/collove/pasteme-cli/) to paste their snippets. It's the only way for now.
